keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigControllerTests.kt

```diff
@@ -279,8 +282,8 @@ internal class DeliveryConfigControllerTests : JUnit5Minutests {
         derivedContext<ResultActions>("the submitted manifest is missing a required field") {
           fixture {
             val mapper = when (contentType) {
-              APPLICATION_YAML -> configuredYamlMapper()
-              else -> configuredObjectMapper()
+              APPLICATION_YAML -> yamlMapper
+              else -> jsonMapper
```

Comment:
This didn't ultimately matter, but I think it makes sense to use the injected Spring bean mappers here rather than creating new ones